### DISCUSSION TITLE
hi I am beginner of python and github .  I Just wonder about how i use open source

### DISCUSSION THREAD

**Discussion by woaks59** [community/community#22285] [New to GitHub]
<p>Hi I am too much beginner of github and python. </p>
<p>so i wonder about how i use open source code in python</p>
<p>"<a href="https://github.com/seroot/Glove" rel="nofollow noopener">https://github.com/seroot/Glove</a> " i got source of python in there</p>
<p>so which file that i have to download to using this code?</p>

**Comment by eyarz** [comment]
<p>It is hard to give you the right answer because some data is missing, so I will try to point you in the right direction -</p>
<p>The best practice is to download and install opensource code is via the package manager (for python it is <a href="https://pypi.org/project/pip/" rel="nofollow noopener">pip</a>).</p>
<p>I think you are missing the basic of programming with python, so I would recommend you to take a basic online course :) </p>
<p>After catching up, come back and ask again if it will still be relevant.</p>

**Comment by Wabri** [comment, marked as answer]
<p>Hi <a class="mention" href="/woaks59">@woaks59</a>,</p>
<p>Welcome to Github Community Forum!</p>
<p>To use that repository you can run the Main.py python file inside the root of the project. To do that you need to know: what dependencies the project have and what python version are used. I search for the requirements.txt but it doesn’t exists so you need to do some tries.</p>
<p>Try to run the Main.py:</p>
<pre><code>python Main.py
</code></pre>
<p>and if there are some dependencies not satisfied try to install with pip and hope the best.</p>
<p>For example if the <em><strong>xyz</strong></em> dependency is not satisfy, try to run this command:</p>
<pre><code>pip install xyz
</code></pre>
<p>Obviously you need to have pip and also python installed in your system, see the official pages: <a href="https://www.python.org/" rel="nofollow noopener">python</a>, <a href="https://pip.pypa.io/en/stable/" rel="nofollow noopener">pip</a>.</p>
<p>Hope to solve your problem :blush: </p>
<p>-Gabriele-</p>
<p>Mark helpfull posts with Accept as Solution to help other users locate important info. Don’t forget to give Kudos for great contents!</p>

**Comment by woaks59** [comment]
<p>Hi Gabriele</p>
<p>First Thanks to answer my question !!</p>
<p>Now I am trying to learn about python now </p>
<p>and your answer make me to reach easy way to learn about python! </p>
